src/platform/web/lib/util/webstorage.ts

```diff
@@ -14,16 +16,16 @@ class Webstorage implements IWebStorage {
      * somewhat roundabout way. (If unsupported or no global object,
      * will throw on accessing a property of undefined) */
     try {
-      global.sessionStorage.setItem(test, test);
```

Comment:
How was this even working, given that `global` doesn't exist on the web?

Reply:
Because webpack scans for `global` access and replaces it depending on the platform it compiles for